src/gui/core/window_builder.cpp

```diff
@@ -70,11 +70,11 @@ builder_widget_lookup()
  * be tuned. This page will describe what can be tuned.
  *
  */
-twindow* build(CVideo& video, const twindow_builder::tresolution* definition)
+window* build(CVideo& video, const builder_window::window_resolution* definition)
```

Comment:
I'm suggesting to rename the function. And now, the other functions too in order to be verbified :)